src/js/setup.js

```diff
@@ -72,7 +72,8 @@ var autoSetup = function(){
 
 // Pause to let the DOM keep processing
 var autoSetupTimeout = function(wait, vjs){
-  videojs = vjs;
+  if (vjs)
```

Comment:
our style guide requires to add curly braces around all blocks, but I can just add them during the merge.